hotel-management/src/pages/index.tsx

```diff
@@ -0,0 +1,7 @@
+export { CartPage } from '@/pages/CartPage';
+export { EntryPage } from '@/pages/EntryPage';
+export { ErrorPage } from '@/pages/ErrorPage';
+export { HomePage } from '@/pages/HomePage';
+export { RoomDetailsPage } from '@/pages/RoomDetailsPage';
+export { RoomFormPage } from '@/pages/RoomFormPage';
```

Comment:
These alias paths can cause circular dependencies, please help to research about this. And for the solution for this is if import in the same alias path, for example here is `@/pages` please use the relative path `../`